hack/docker.sh

```diff
@@ -0,0 +1,26 @@
+#! /bin/bash
```

Comment:
to double check, does this script work on Linux?

Reply:
yep, I've tested it in Debian